tRecorderApi/config.py

```diff
@@ -0,0 +1,4 @@
+bind = ['172.19.145.88:8000']
```

Comment:
Generally speaking, config files should be excluded from a repo.  I'll let this slide since there isn't any sensitive info.